src/common.cpp

```diff
@@ -2703,7 +2703,7 @@ uint32_t get_version()
 
 void get_data_lang_path(PCHAR buffer)
 {
-	strcpy(buffer, basedir);
+	strcpy(buffer, ff8 ? ff8_externals.app_path : basedir);
```

Comment:
Where is `app_path` configured out of curiosity? And how does it different from `basedir` in this case?

Reply:
app_path is configured via the register key "AppPath". So in the Steam version, it comes from "dotemuRegQueryValueExA" override, which is by default `GetCurrentDirectory() + \`.
Then from this value, the game build paths, like {app_path}\data\field.fs for example.

FFNx does have an option to override app_path, after the RegQueryValueExA, but today it only works for the 2000-version, because we don't always use app_path where we should for game data.

(FF7 have something similar, but relative to "data" dir, but I don't want to mess with 7th Heaven)

`basedir` is `GetCurrentDirectoryA()`, which is similar to the default from the Steam version, but this is not overridable. We use this value for mods, config or data used by FFNx, not for internal game data (I think).

Reply:
With the 2000 version, I'm able to install FFNx in `C:/Users/myst6re/Documents/FF8`, put the FF8.exe and the bink dll, and run the game installed to `C:/Program Files/Square Soft, Inc/FINAL FANTASY VIII` without moving all the data, simply by setting app_path to `C:/Users/myst6re/Documents/FF8`

It has a huge advantage: I don't have to mess with admin rights, and I can configure that faster than what for example Satsuki as to do in its pack for ff7